src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -72,6 +72,27 @@ public function getDefinition(): FixerDefinitionInterface
                         'keep_multiple_spaces_after_comma' => false,
                     ]
                 ),
+                new CodeSample(
+                    "<?php\nfunction sample(#[Foo] #[Bar] \$a=10,\n    \$b=20,\$c=30) {}\nsample(1,  2);\n",
+                    [
+                        'on_multiline' => 'ensure_fully_multiline',
+                        'attribute_placement' => 'ignore',
+                    ]
+                ),
+                new CodeSample(
+                    "<?php\nfunction sample(#[Foo] #[Bar] \$a=10,\n    \$b=20,\$c=30) {}\nsample(1,  2);\n",
```

Comment:
I believe there should be new line somewhere between attributes so the example would show how `same_line` works. There is test case that ensures it works correctly, but this code sample does not show it 🙂.